packages/backend/src/utils/parsers/quadlet-dryrun-parser.spec.ts

```diff
@@ -106,11 +106,11 @@ test('should parse stderr properly and properly set state', async () => {
 
   expect(container.path).toBe('/home/user/.config/containers/systemd/nginx.container');
   expect(container.state).toBe('error');
-  expect(container.content).toBeUndefined();
+  expect(container.service).toBeUndefined();
```

Comment:
suggestion(non-blocking): maybe use `isServiceQuadlet`


```suggestion
  expect(isServiceQuadlet(container)).toBeFalsy();
```